pact/Pact/Core/IR/Eval/CoreBuiltin.hs

```diff
@@ -924,29 +956,38 @@ coreReadString :: (CEKEval step b i m, MonadEval b i m) => NativeFunction step b
 coreReadString info b cont handler _env = \case
   [VString s] -> do
     viewEvalEnv eeMsgBody >>= \case
-      PObject envData ->
+      PObject envData -> do
+        chargeGasArgs info $ GObjOp $ ObjOpLookup s $ M.size envData
```

Comment:
Ditto, maybe we don't need to charge simply for reading a field from env data. Parsing is fine, env data can't ever grow to thousands and thousands of fields.

Reply:
Resolved. I think this is actually okay now after talking to Georg.